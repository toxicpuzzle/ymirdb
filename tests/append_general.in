SET a 1
SET b 1
SET c 1
SET d 1
SET a 1 b c d
LIST ENTRIES
BACKWARD b
SET e 1
FORWARD a
APPEND a e
FORWARD a
BACKWARD e
APPEND a c d 
FORWARD a
LIST ENTRIES
DEL e
LIST ENTRIES   
DEL d
LIST ENTRIES
DEL a
LIST ENTRIES
bye