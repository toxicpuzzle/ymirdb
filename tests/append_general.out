> ok

> ok

> ok

> ok

> ok

> d [1]
c [1]
b [1]
a [1 b c d]

> a 

> ok

> b c d 

> ok

> b c d e 

> a 

> ok

> b c d e 

> e [1]
d [1]
c [1]
b [1]
a [1 b c d e c d]

> not permitted

> e [1]
d [1]
c [1]
b [1]
a [1 b c d e c d]

> not permitted

> e [1]
d [1]
c [1]
b [1]
a [1 b c d e c d]

> ok

> e [1]
d [1]
c [1]
b [1]

> bye